tests/test_update.py

```diff
@@ -113,6 +113,7 @@ def test_broken_image_update(self, bitbake_variables, connection):
             assert output.find(active_before) >= 0
             assert output.find(passive_before) < 0
 
+    @pytest.mark.cross_platform
```

Comment:
Not this one. This is testing the boot loader integration, which we need to keep.